src/modules/SlideShowCallbacks.rb

```diff
@@ -310,21 +318,47 @@ def DisplayStartInstall(pkg_name, pkg_location, pkg_description, pkg_size, delet
                 Builtins.substring(part, 0, 1) != "/"
               part = Ops.add("/", part)
             end
+
+            # nothing to install on this partition, skip it (bsc#926841)
+            if required_space <= 0
+              log.debug("Nothing to install at #{part}, skipping")
+              next
+            end
+
+            target_dir = File.join(Installation.destdir, part)
+
+            # handle missing directories (not existing yet or incorrect metadata),
```

Comment:
Not sure if this can happen, but please consider this scenario:

During installation, the target filesystems are mounted below /mnt; if any separate filesystem on the target could not be mounted, so that directory isn't there, will it move up the hierarchy to / which is the inst-sys root filesystem which is read-only or has very little available space. Can this cause a problem here?

Reply:
If the target directory below `/mnt` is not there then it should stop at `/mnt`. If `/mnt` is not there than a wrong disk space check would be the least problem you had, the system would be so broken that you cannot continue anyway...